### ISSUE TITLE
Specify section

### ISSUE THREAD

**Issue by hinxx** [meronz/vscode.manpages#29]
When trying to open a man page for `open` is get the man page for `xdg-open` in section 1.
The `open` is in section 2 and in CLI one needs to do `man 2 open`.
AFAIK, there are lots of other similar cases that require one to explicitly specify a section of the man pages.

How can I do the same with this extension?

**Comment by meronz** [comment]
You just need to select the whole word (xdg-open) and rightclick > Man: open page. You don't need to manually select the words. On my machine works as intended, if I place the cursor on xdg-open, the man page for xdg-open is shown.

**Comment by meronz** [comment]
I can understand that the wording of the menu labels was imprecise, so I changed them in 0.1.5.
Try the new release once it lands on the store, and feel free to open a new issue if the problem is not resolved :)

Related PR: https://github.com/meronz/vscode.manpages/pull/31

**Comment by hinxx** [comment]
> if I place the cursor on xdg-open, the man page for xdg-open is shown.

Does it open 'open' man page when you place a cursor on word "open"? I get a man page for xdg-open, which is not what I wanted.

**Comment by meronz** [comment]
- Are you using the latest extension version? (0.1.5)
- Did you configure any option or argument?
- If you try to search for the word (In command palette type "Man: open page", and type "open"), does it work correctly?

**Comment by hinxx** [comment]
Using the latest version, AFAICT, yes. I did not configure the extension with any options/arguments. The same is observed if I use the command palette.
I've recorded a small video that might help.

https://github.com/meronz/vscode.manpages/assets/2725009/31c512f4-1534-4c1c-841c-2f8a106a556b

**Comment by meronz** [comment]
I'll try to replicate your issue and get back to you. Thanks for the video!

**Comment by meronz** [comment]
I've added some logging in the latest release (0.1.6). Could you update and get back with the logs?

You can find them in the Output tab of VSCode:
![image](https://github.com/meronz/vscode.manpages/assets/10725286/47261143-655e-4c03-9e04-2c90fa2f8dce)

**Comment by hinxx** [comment]
Here is the video of it looks.

https://github.com/meronz/vscode.manpages/assets/2725009/0ada6883-e50d-464c-95f9-5760f941abf8

I see that the "correct" argument is passed to `man` command. But it seems to be the `man` feature on which man page is opened. For example if I type `man open` in terminal I get the `xdg-open` man page, too, instead of `open`.

I noticed another issue. When a single argument function call with a variable argument is under the cursor extension takes the wrong text; ie. for `strdup(orig);` it calls `man  orig strdup`. But tinkering further with this lead to a solution for the original `open` problem. If I type `open(2);`, hover the 'open' and try to open the man page I get what I expected; the extension calls `man 2 open` and man page for `open` in section 2 is opened. In CLI I could do `man 'open(2)'` to achieve the same (which seems like what is happening here?).

**Comment by hinxx** [comment]
I meant to suggest that `-s 2,3` could added to the `man` arguments in the extension setting to avoid looking at the section 1 since that is not applicable to C/C++. But then I reckoned that the extension could be used with other programming/scripting languages like bash. I made a test with a bash script to discover that in this case just hovering and clicking on the `xdg-open` results in no man page being opened. But selecting the 'xdg-open' text results in opening the correct man page..

https://github.com/meronz/vscode.manpages/assets/2725009/737c1f06-0320-47ee-be5d-ea376aa2eeee
